This patch relaxes security checks for writing the /sys/class/rfkill/*/state
attribute. Normally it would check for CAP_NET_ADMIN but X92 firmware supposes
it's limited only by filesystem access rights.

diff -ur original/net/rfkill/core.c modified/net/rfkill/core.c
--- original/net/rfkill/core.c	2017-05-06 16:52:34.845945449 +0300
+++ modified/net/rfkill/core.c	2017-05-06 16:52:34.845945449 +0300
@@ -703,8 +703,10 @@
 	unsigned long state;
 	int err;
 
+/*
 	if (!capable(CAP_NET_ADMIN))
 		return -EPERM;
+*/
 
 	err = kstrtoul(buf, 0, &state);
 	if (err)
